CMakeLists.txt

```diff
@@ -123,6 +139,11 @@ if(USE_GPU)
     ADD_DEFINITIONS(-DUSE_GPU)
 endif(USE_GPU)
 
+if(__OPENCL_PYTHON_PACKAGE)
+    include(CMakeOpenCLPythonPackage.txt)
```

Comment:
since this is a `cmake` script file, could you change the extension to `cmake`? similar to https://github.com/microsoft/LightGBM/blob/master/R-package/src/cmake/modules/FindLibR.cmake

Reply:
Renamed.